docs/tutorials/access-service-across-clusters.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Access service acrosss clusters within native service name
+---
+
+In Karmada, the MultiClusterService can enable users to access services across clusters with the native service name, like `foo.svc`, with the aim of providing a seamless experience for users to access services across clusters.
+
+This document provides an example of how to enable MultiClusterService for accessing service across clusters with native service name.
+
+## Prerequisites
```

Comment:
Should we explain the limitations of using this feature in this chapter, such as the requirement for interconnectivity of container networks between member clusters?